app/views/team.py

```diff
@@ -1,23 +1,32 @@
-from flask import render_template, Blueprint, redirect, url_for, request
+from flask import render_template, Blueprint, redirect, url_for, request, flash
 from flask_login import login_required
 
-from app.forms import RegistrationForm
+from app.forms import EditUserForm, RegistrationForm
 from app.models import User
 
+from app.logger import log
+
 team_blueprint = Blueprint("team", __name__)
 
 
 @team_blueprint.route("/edit_card/<int:user_id>", methods=["POST"])
 @login_required
 def edit_card(user_id):
-    form = RegistrationForm(request.form)
-    user = User.query.get(user_id)
-    user.username = form.username.data
-    user.email = form.email.data
-    user.position = form.position.data
-    user.phone = form.phone.data
-    user.password = form.password.data
-    user.save()
+    form = EditUserForm(request.form)
```

Comment:
I think "request.form" set as a default value in "EditUserForm" class

Reply:
yes